### ISSUE TITLE
training resnet-164 in cifar10 — train accuracy

### ISSUE THREAD

**Issue by blazerygy** [dmlc/mxnet#5914]
I want to train resnet-164 in cifar10.
If I use one machine to train (the batch size is 128) ,  the train accuracy is **48%** at the end of the first epoch, but if I use two machine to train (kvstore = dist_sync, the batch size is 64 on each machine) , the train accuracy is  only **43% (not 48%)** at the end of the first epoch,  why?
because of the loss of gradients precision? anyone can explain it? thx

**Comment by tornadomeet** [comment]
use batch-norm layer?

**Comment by blazerygy** [comment]
@tornadomeet how to use batch-norm layer? I just follow your introductions in `https://github.com/tornadomeet/ResNet`
'python -u train_resnet.py --data-dir data/cifar10 --data-type cifar10 \
  --depth 164 --batch-size 128 --num-examples 50000 --gpus=0,1'
thx

**Comment by tornadomeet** [comment]
because resnet has used bn layer, and bn layer will not be the same when using multi-gpu.

**Comment by tornadomeet** [comment]
i mean your issue is normal, not a problem.

**Comment by blazerygy** [comment]
@tornadomeet thx, i got it
